packages/annotorious-svelte/vite.config.js

```diff
@@ -24,15 +24,17 @@ export default defineConfig({
       fileName: (format) => `annotorious-svelte.${format}.js`
     },
     rollupOptions: {
-      external: [...Object.keys(packageJson.peerDependencies)],
+      external: [
+        ...Object.keys(packageJson.peerDependencies),
+        '@annotorious/core',
+        '@annotorious/annotorious',
+        '@annotorious/openseadragon'
+      ],
```

Comment:
I think in this case, this might be ok. Because we're only publishing an ES package here?

Reply:
Exactly! Reference to the code from those packages will be left untouched and we'll just have a bunch of `import ... from '@annotorious/...'` at the top of the resulting bundle